rocAL/rocAL/source/meta_data/bounding_box_graph.cpp

```diff
@@ -26,6 +26,7 @@ void BoundingBoxGraph::process(pMetaDataBatch input_meta_data, pMetaDataBatch ou
     for (auto &meta_node : _meta_nodes)
     {
         meta_node->update_parameters(input_meta_data, output_meta_data);
+        input_meta_data = output_meta_data->clone();
```

Comment:
Have we tested this with other trainings too ?

Reply:
Ran classification and mask trainings with this change, @fiona-gladwin can u comment on the SSD training perf with this change here?

Reply:
Ran Retinanet training:
CPU backend : Not much affected with this change
GPU backend : In progress